modules/contact/test/tests/3d-mortar-contact/tests

```diff
@@ -108,4 +108,26 @@
     allow_warnings = true
     max_parallel = 1
   []
+  [frictional-mortar-3d-penalty]
+    type = 'Exodiff'
+    input = 'frictional-mortar-3d-penalty.i'
+    min_ad_size = 100
+    exodiff = 'frictional-mortar-3d-penalty_out.e'
+    requirement = 'The system shall solve a 3D frictional problem with mortar '
+                  'constraint using a penalty approach'
+    rel_err = 1.0e-5
+    abs_zero = 1.0e-5
+    max_parallel = 8
```

Comment:
The reason I initially added that setting is because it has a small number of finite elements, so it may not make sense to run it with a lot of processors. I will remove it.

Reply:
It may not make sense from a performance standpoint, but it's good for test robustness if it can run with larger processor counts

Reply:
Makes sense. 

At this point I am confused as to whether in setting, for this test in particular, has any practical meaning. I know we can set `min_parallel` and force CIVET to employ that number of processors, but without it, wouldn't it just do a two-way run under a parallel module recipe?

Reply:
You mean `-p2` ? On CIVET this recipe may only be run with 2 processes ... but some day we may add a parallel sweep recipe for modules or contact specifically. Additionally a user may run with something other than `-p2`

Reply:
> You mean -p2 ? 

Yes.